template/page-home.html.jinja

```diff
@@ -38,37 +38,37 @@
+      <h6 class="longform mb-10">{% trans %}The XRP Ledger (XRPL) is a decentralized, public blockchain led by a global community of businesses and developers looking to solve problems and create value.{% endtrans %}</h6>
+      <p class="mb-0">{% trans %}Proven reliable over more than a decade of error-free functioning, the XRPL offers streamlined development, low transaction costs, high performance, and sustainability. So you can build with confidence–and move your most critical projects forward.{% endtrans %}</p>
     </div>
   </section>
 
   <section class="container-new py-26">
     <div class="d-flex flex-column-reverse col-sm-8 p-0">
       <h3 class="h4 h2-sm">{% trans %}Why developers choose the XRP Ledger{% endtrans %}</h3>
-      <h6 class="eyebrow mb-3">{% trans %}Discover Benefits{% endtrans %}</h6>
+      <h6 class="eyebrow mb-3">{% trans %}Benefits{% endtrans %}</h6>
     </div>
     <ul class="mt-10 card-grid card-grid-3xN" id="benefits-list">
       {% set cards = [
         { "id": "public",
-          "title": _("Public and Decentralized Structure"),
+          "title": _("Public and Decentralized"),
           "description": _("Open source, open to anyone to build on, maintained by the community")},
         { "id": "streamlined",
-          "title": _("Streamlined <br class='until-sm'/>Development"),
-          "description": _("Tools and documentation that speed development and reduce time to market")},
+          "title": _("Streamlined Development"),
+          "description": _("Intentional innovations, tools and documentation reduce time to market")},
         { "id": "performance",
-          "title": _("High <br class='until-sm'/>Performance"),
-          "description": _("Capable of settling thousands of transactions in seconds")},
+          "title": _("High Performance"),
+          "description": _("Thousands of transactions settled in seconds")},
         { "id": "low-cost",
           "title": _("Low Cost"),
-          "description": _("At fractions of a penny per transaction, costs are inexpensive enough to enable a wide variety of use cases")},
+          "description": _("At fractions of a penny per transaction, the XRPL can fuel expansive use cases")},
         { "id": "community",
-          "title": _("Vibrant Community"),
-          "description": _("Developers, validators, users, and businesses make the XRP Ledger better every day")},
+          "title": _("Motivated Community"),
+          "description": _("Companies, developers, validators, and users work together to make the XRP Ledger better every day")},
         { "id": "reliability",
           "title": _("Proven Reliability"),
-          "description": _("8+ years of consistent performance over more than 63 million ledgers")},
+          "description": _("10+ years of error-free, uninterrupted performance over more than 63 million ledgers")},
```

Comment:
```suggestion
          "description": _("10+ years of error-free performance over more than 80 million ledgers")},
```
- Not sure how "uninterrupted" is defined, but there was a short ~15 minute pause in forward progress in Nov 2021.
- Though the first 32570 ledgers were lost, the network has now closed and fully validated more than 82,090,000 [ledgers](https://livenet.xrpl.org/)